### PR TITLE
Add support when building iOS target

### PR DESCRIPTION
**Pull request by kylehickinson**
iOS `relic_conf.h` based off of build args provided for iOS build: https://github.com/relic-toolkit/relic/blob/master/preset/armios-pbc-112.sh

Simulator uses the macOS include since simulator runs on macOS architecture.

### CODE REVIEW

File: anon/ios_include/relic_conf.h [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+/**
+ * @file
+ *
+ * Project configuration.
+ *
+ * @version $Id: relic_conf.h.in 45 2009-07-04 23:45:48Z dfaranha $
+ * @ingroup relic
+ */
+
+#ifndef RELIC_CONF_H
```
Comment (bridiver) [inline anon/ios_include/relic_conf.h]: this is the same as the mac version? Are we sure no changes are needed for ios?

Comment (kylehickinson) [reply anon/ios_include/relic_conf.h]: There's no option to support iOS (see `OPSYS`) but iOS is based on macOS so it should in theory be fine.

I'm not familiar with anonize, are their tests I can run to ensure it is correct?

Comment (kylehickinson) [reply anon/ios_include/relic_conf.h]: I actually found this: https://github.com/relic-toolkit/relic/blob/master/preset/armios-pbc-112.sh

Going to update this

**Comment by kylehickinson** [comment]
I had updated the gen based on relic's own iOS preset found here: https://github.com/relic-toolkit/relic/blob/b984e901ba78c83ea4093ea96addd13628c8c2d0/preset/armios-pbc-112.sh#L12

**Pull request merged**